src/generation_tools.jl

```diff
@@ -1,6 +1,5 @@
 ## generation_tools.jl
-export ϕ_tuple, idx_to_dim
-
+export ϕ_tuple, idx_to_dim 
```

Comment:
shall we delete them or not?

Reply:
We should leave their status as it previously was.